janitor/functions/read_commandline.py

```diff
@@ -0,0 +1,77 @@
+import pandas_flavor as pf
+import pandas as pd
+from subprocess import Popen, PIPE
+import sys
+
+
+@pf.register_dataframe_method
+def read_commandline() -> pd.DataFrame:
+    """
+    Read command from commandline.
+
+    argv[0] should be program name
+    argv[1:] will be the a list of commands available to the shell
+    such as:
+    cat sep-quarter.csv | grep .SEA1AA
+
+    calling
+    df = read_commandline()
+    will create a new dataframe from the commandline arguments given.
```

Comment:
```suggestion
    In this case, you can use the following Python code to load the dataframe:
    
    ```python
    import janitor as jn
    df = jn.read_commandline("cat sep-quarter.csv | grep .SEA1AA")
    ```
```